client-ui/src/app/components/lineage/lineage-details/operation-properties-details/properties/properties.component.ts

```diff
@@ -36,15 +35,18 @@ export class PropertiesComponent implements OnDestroy {
   @Input()
   propertyType: string
 
+  @Input()
+  propertyName: string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'propertyName' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4585293)

Reply:
let's specify visibility explicitly. It helps.